apps/web/modules/onboarding/components/plan-icon.tsx

```diff
@@ -0,0 +1,107 @@
+import classNames from "@calcom/ui/classNames";
+import { Icon, type IconName } from "@calcom/ui/components/icon";
+
+export function PlanIcon({ icon, variant = "single" }: { icon: IconName; variant?: "single" | "double" }) {
+  const renderIconContainer = (index: number) => {
+    // For double variant: 55px icon width + 24px gap = 79px between centers, so ±39.5px from center
+    const leftPosition = variant === "double" ? `calc(50% + ${index === 0 ? -39.5 : 39.5}px)` : "50%";
+
+    return (
+      <div
+        key={index}
+        className="bg-default absolute top-[10px] flex h-[55px] w-[55px] -translate-x-1/2 items-center justify-center overflow-clip rounded-full"
+        style={{
+          left: leftPosition,
+          background: "linear-gradient(to bottom, var(--cal-bg, #ffffff), var(--cal-bg-muted, #f7f7f7))",
+          boxShadow:
+            "0px 2.818px 5.635px 0px var(--cal-border-subtle), 0px 0px 0px 0.704px var(--cal-border), 0px 0.704px 3.522px -2.818px rgba(0, 0, 0, 0.3)",
+        }}>
+        {/* Icon */}
+        <div className="size-8 flex items-center justify-center">
+          <Icon name={icon} size={24} strokeWidth={1.75} className="text-emphasis opacity-80" />
+        </div>
+
+        {/* Inner highlight/shine effect */}
```

Comment:
Leaving these comments in here because its actually pretty helpful if you ever need to touch this file. Lots of madness going on with css